### DISCUSSION TITLE
Release Candidate One of 3.11.0 is out

### DISCUSSION THREAD

**Discussion by pljones** [jamulussoftware/jamulus#3371] [Announcements]
We have tagged the first release candidate for Jamulus 3.11.0, [Jamulus 3.11.0rc1](https://github.com/jamulussoftware/jamulus/releases/tag/r3_11_0rc1).

There are signed macOS dmg files using the same bundle identifier as beta2 but not earlier versions.  If you were not on beta2 you will - as you would if you wait for the final release - need to recreate your local configuration or move it [as documented here](https://jamulus.io/wiki/Software-Manual#backing-up-jamulus).  The new identifier is `app.jamulussoftware.Jamulus`.

Please download the installer for your platform from the [release note](https://github.com/jamulussoftware/jamulus/releases/tag/r3_11_0rc1) and start using it for your normal usage of Jamulus - **Keep a back-up of your 3.10.0 install** for now.

If you find any problems, please raise a bug report making it clear that you're using the **3.11.0rc1** release.

For people who have to build from source, please note the dependencies have changed since 3.10.0 - the one major change being the minimum version of Qt is now 5.12.2.  (Builds have been with Qt up to 6.7.2.)

The updated website will be release formally once translations are reviewed, complete and the final release is announced.

Thanks to all the contributors and application translators: 
@AdamGLIN, @ann0see, @BLumia, @ciampix, @comradekingu, @danryu, @genesisproject2020, @gilgongo, @hoffie, @houbsta, @jose1711, @jujudusud, @mansil, @melcon, @MisterCosta96, @pljones, @rdica, @softins, @trebmuh, Henk De Groot, ignotus, taitep

**Comment by rdica** [comment]
Thanks :+1: 
Compiled client on Ubuntu Studio 22.04, servers on Ubuntu 22.04/24.04, beginning testing.

**Reply by rdica** [reply]
Client's working fine on Ubuntu Studio 22.04. No issues seen with server on Ubuntu 22.04/24.04.

**Comment by ann0see** [comment]
macOS signed build runs.

**Comment by dcorson-ticino-com** [comment]
Windows client working fine.

**Comment by pljones** [comment]
Jamulus release 3.11.0 is now out.  The update notification should follow shortly.

**Reply by ann0see** [reply]
See https://github.com/orgs/jamulussoftware/discussions/3377 for the final release
